python/ray/util/xgboost/simple_tune.py

```diff
@@ -33,6 +34,19 @@ def train_model(config):
 # __train_end__
 
 
+# __load_begin__
```

Comment:
does this show up in the docs properly?

Reply:
This file is actually not used in the docs -- a code block is used instead 

https://github.com/ray-project/ray/blob/master/doc/source/xgboost-ray.rst#hyperparameter-tuning